lib/stellar_core_commander/commander.rb

```diff
@@ -18,15 +18,23 @@ def initialize(process_type, process_options={})
       @processes = []
     end
 
-    Contract Transactor, Symbol, ArrayOf[Symbol], Num => Process
+    Contract Transactor, Symbol, ArrayOf[Symbol], Num, Bool => Process
     #
     # make_process returns a new, unlaunched Process object, bound to a new
     # tmpdir
-    def make_process(transactor, name, quorum, thresh)
+    def make_process(transactor, name, quorum, thresh, manual_close=false)
```

Comment:
Just as a heads up, this might also become an options hash once I start working on multi-docker host stuff.

Another option I've been considering is some sort of auxiliary file (or section) that defines how each process name should be spawned, separate from the recipe's `process` declaration.